app/components/standings/standings.scss

```diff
@@ -0,0 +1,146 @@
+@import 'variables';
+
+
+header {
```

Comment:
to clarify. Everything in the page's css file should be wrapped in a page ID. that way changed to this page dont affect anything else. These scss files should be pretty small since most page content should follow the global styling in main